shared/js/actions.js

```diff
@@ -145,6 +145,14 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Notifies that a tile has been received from the other peer.
+     */
+    ReceivedTile: Action.define("receivedTile", {
+      tile: String
+      // sentTimestamp: String (optional)
```

Comment:
The types are checked when an action is created. Your tests only pass in `String`s.